README.md

```diff
@@ -20,10 +20,10 @@ Description
 Usage
 =====
 
-1. Go to APIscrape/APIscrape.php and edit the value of constants DBNAME, HOST, USER and PASSWORD to your 
+1. Go to src/DBconnector.php and edit the value of constants DBNAME, HOST, USER and PASSWORD to your 
```

Comment:
why? this is wrong, the api scrape does not use DBconnector.php

Reply:
done